print.go

```diff
@@ -6,8 +6,17 @@ import (
 	"github.com/koyachi/go-term-ansicolor/ansicolor"
 )
 
-func Header() {
-	fmt.Println(ansicolor.Cyan("Looper 0.3.2 is watching your files"))
+func Header(gtargs []string) {
+	fmt.Println(ansicolor.Cyan("Looper 0.3.4 is watching your files"))
+
+	if len(gtargs) > 0 {
+		fmt.Printf(ansicolor.Green("Passing %d addition argument(s) to go test:\n"), len(gtargs))
```

Comment:
It can be removed.  It was mostly a leftover from the first commit that still had differing `looper`/`gotest` arguments so it was clear that your arguments were indeed getting picked up (ie: a debugging artifact).

Now that they are all being passed to go test I see no reason to keep this output, which would also revert the method signature to be the same as it was before I tinkered with it.